src/App.js

```diff
@@ -1,12 +1,24 @@
 import React, { Component } from 'react';
+import './app.scss';
+
+import { Button } from 'carbon-components-react';
+import { Content } from 'carbon-components-react/lib/components/UIShell';
+import TutorialHeader from './components/TutorialHeader';
+
+import { Route, Switch } from 'react-router-dom';
+import LandingPage from './content/LandingPage';
+import RepoPage from './content/RepoPage';
 
 class App extends Component {
   render() {
     return (
-      <div>
-        Hello Carbon! Well, not quite yet. This is the starting point for the
-        Carbon tutorial.
-      </div>
+      <>
+        <TutorialHeader />
+        <Content>
```

Comment:
You're missing the `<Switch>` component. See https://www.carbondesignsystem.com/tutorial/react/step-1#add-routing